### PR TITLE
Fix a typo.

### PR DESCRIPTION
**Pull request by BeastGamer81**
Manager User --> Manage User

### CODE REVIEW

**Comment by Jelcoo** [comment]
Please do not merge into a release branch directly. Change your target branch to develop or 1.0-develop

**Comment by BeastGamer81** [comment]
> Please do not merge into a release branch directly. Change your target branch to develop or 1.0-develop

There you go.

File: config/app.php
Comment (PadowYT2) [inline config/app.php:1]: Remove this change

**Review by PadowYT2 — COMMENTED** [review]
You must push to develop-1.0, the admin on develop isn't Blade

**Pull request closed without merging**